.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Cilium",
-  "image": "quay.io/cilium/cilium-builder:2ad400e76cf46e3e8d13c1962f25af09c0a1fef1@sha256:b2d188445e47a5abfe2649ada18371efa427cbc34f2f968b84a5235bbff4d47a",
+  "image": "quay.io/cilium/cilium-builder:f11ed10c9ff739cfdfdd150013da5c5a98d30555",
```

Comment:
should we have sha component here as well?

Reply:
Yep, things are not going as expected, and this my 1st time dealing with this, so any feedback/help is super welcome 🙏🏼  According to Fabio:

> In v1.15, the first "Base Image Lint" started to fail from https://github.com/cilium/cilium/commit/81602f331c83c20108c1273d0bcef847196e1ceb and running `make -C images/ update-builder-image` from v1.15 HEAD is not working: https://github.com/cilium/cilium/pull/38748.
I think the multiple failures for all the subsequent v1.15 commits prevented the builder image (that should have been updated again in one of the subsequent commit) to be pushed to [quay.io](http://quay.io/), so the update-builder-image fails to get the image sha.


What's the best way to fix this?

Reply:
I was waiting for the CI build last week, and forgot. It should be fine now,